src/_main_/utils/constants.py

```diff
@@ -52,7 +52,7 @@
 
 STATES = load_json(BASE_DIR + "/database/raw_data/other/states.json")
 
-ME_LOGO_PNG = "https://www.massenergize.org/wp-content/uploads/2021/07/cropped-me-logo-transp.png"
+ME_LOGO_PNG = "https://www.massenergize.org/wp-content/uploads/2023/03/MassEnergize-logo.png"
```

Comment:
@abdullai-t Can't we pick this from param store or .env ?

Reply:
We could push it to env